ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -79,21 +136,36 @@ export default function FeeCard ({
           </div>
         </div>
         {!hideTokenApprovalRow && (
-          <div className="fee-card__top-bordered-row">
+          <div className="fee-card__row-header">
             <div className="fee-card__row-label">
               <div className="fee-card__row-header-text">
                 {t('swapThisWillAllowApprove', [tokenApprovalTextComponent])}
               </div>
-              <div className="fee-card__link" onClick={() => onTokenApprovalClick()}>
-                {t('swapEditLimit')}
+              <div className="fee-card__info-tooltip-container">
+                <InfoTooltip
+                  position="top"
+                  contentText={t('swapEnableDescription', [tokenApprovalSourceTokenSymbol])}
+                />
               </div>
+            </div>
+            <div className="fee-card__link" onClick={() => onTokenApprovalClick()}>
+              {t('swapEditLimit')}
+            </div>
+          </div>
+        )}
+        <div className="fee-card__top-bordered-row">
+          <div className="fee-card__row-label">
+            <div className="fee-card__row-header-text">
+              {t('swapQuoteIncludesFee', [metaMaskFee])}
+            </div>
+            <div className="fee-card__info-tooltip-container">
               <InfoTooltip
                 position="top"
-                contentText={t('swapEnableDescription', [tokenApprovalSourceTokenSymbol])}
+                contentText={t('swapMetaMaskFeeDescription')}
```

Comment:
missing fee variable which will resolve @darkwing 's comment about the fees being different. The $1 is the replacement placeholder in our i18n solution. Do we have a checker for these types of issues?

Reply:
> Do we have a checker for these types of issues?

You mean as part of our build process? No we don't, but there is an error that gets rendered to the console in these cases.